src/components/form-elements/text-input/TextInput.js

```diff
@@ -60,11 +60,11 @@ class TextInput extends React.Component<Props, State> {
         };
     }
 
-    UNSAFE_componentWillReceiveProps(nextProps: Props) {
+    componentDidUpdate(prevProps: Props) {
```

Comment:
Could destructure out the value from prevProps

Reply:
Is that the preferred option? I don't think the code is as read-able de-structuring from the params directly.

Reply:
We usually suggest folks destructure in the function signature, though it can get hairy with multiple object parameters, such as `prevProps, prevState`.